### DISCUSSION TITLE
How can I use the wireguard addon as client?

### DISCUSSION THREAD

**Discussion by egandro** [hassio-addons/addon-wireguard#145] [Q&A] [answered]
# Problem/Motivation

I want to connect via home assistant to my rootserver on the internet (basically using WG as client).

## Expected behavior

I didn't find any documentation on this. The Alpine tutorial won't work. After boot the /etc/wireguard things are gone.

**Comment by frenck** [comment, marked as answer]
Nope

**Comment by egandro** [comment]
I found this one. https://github.com/bigmoby/addon-wireguard-client

Thx for your help.
